docs/third-party-cdn.md

```diff
@@ -0,0 +1,60 @@
+# Third Party CDN Usage
+
+In some situations it may be required to use your own CDN to serve the Altis powered website or service. Below are some examples that may necessitate this:
+
+- Many web properties are served from a single domain name, whereby you need to control routing. A sub-section of the URL namespace may route to Altis, in which case it's not possible to route all traffic to Altis' CDN for a single domain name.
+- There may be compliance requirements or company-wide mandated use of a specific CDN vendor.
+
+In these cases, Altis Cloud can be configured to serve traffic to your CDN via an origin-pull setup. Altis does not support an origin-push CDN setup. Reach out to Altis Cloud Support for assistance in third party CDN integration as it requires specific manual work by the Altis team.
```

Comment:
Add a link to dashboard support page?

Reply:
Sitenote: It might be worth us adding a `support://` to the documentation module where `support://new` adds the stack name automatically if it can.

Reply:
Ooo yeah, can you add an issue for that?

Reply:
✅  https://github.com/humanmade/altis-documentation/issues/116